expyfun/_experiment_controller.py

```diff
@@ -963,7 +964,13 @@ def _setup_window(self, window_size, exp_name, full_screen, screen_num):
         gl.glShadeModel(gl.GL_SMOOTH)
         gl.glClear(gl.GL_COLOR_BUFFER_BIT)
         v_ = False if os.getenv('_EXPYFUN_WIN_INVISIBLE') == 'true' else True
-        self.set_visible(v_)
+        self.set_visible(v_)  # this is when we set fullscreen
+        # ensure we got the correct window size
+        got_size = win.get_size()
+        if not np.array_equal(got_size, window_size):
+            raise RuntimeError('Requested window size %s but got size %s, '
+                               'is the resolution set incorrectly?'
+                               % (window_size, got_size))
```

Comment:
Can we improve this error message? maybe "Window size requested by config (%s) does not match window size (%s), is the screen resolution set incorrectly?"